src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpApplicationController.cs

```diff
@@ -172,41 +188,57 @@ private async Task<IActionResult> StartApplication(Guid signinId)
                 var allQnaSections = await allQnaSectionsTask;
 
-                var startApplicationRequest = await BuildStartApplicationRequest(qnaResponse.ApplicationId, user.Id, providerRoute, allQnaSequences, allQnaSections);
+                var startApplicationRequest = BuildStartApplicationRequest(qnaResponse.ApplicationId, user.Id, providerRoute, allQnaSequences, allQnaSections);
 
                 var applicationId = await _apiClient.StartApplication(startApplicationRequest);
                 _logger.LogInformation($"RoatpApplicationController.StartApplication:: Checking response from StartApplication POST: applicationId: [{applicationId}]");
 
                 if (applicationId != Guid.Empty)
                 {
                    await SavePreambleInformation(applicationId, applicationDetails);
+                   _logger.LogInformation("Preamble information saved");
 
                     if (applicationDetails.UkrlpLookupDetails.VerifiedByCompaniesHouse)
                     {
                         await SaveCompaniesHouseInformation(applicationId, applicationDetails);
+                        _logger.LogInformation("Companies House information saved");
                     }
 
                     if (applicationDetails.UkrlpLookupDetails.VerifiedbyCharityCommission)
                     {
                         await SaveCharityCommissionInformation(applicationId, applicationDetails);
+                        _logger.LogInformation("Save Charity Commission information saved");
                     }
                 }
+                
+                _logger.LogInformation("StartApplication method completed");
+
+                return applicationId;
             }
 
-            return RedirectToAction("Applications", new { applicationType });
+            return Guid.Empty;
         }
 
-        private async Task<Application.Apply.Start.StartApplicationRequest> BuildStartApplicationRequest(Guid qnaApplicationId, Guid creatingContactId, int providerRoute, IEnumerable<ApplicationSequence> qnaSequences, IEnumerable<ApplicationSection> qnaSections)
+        private StartApplicationRequest BuildStartApplicationRequest(Guid qnaApplicationId, Guid creatingContactId, int providerRoute, IEnumerable<ApplicationSequence> qnaSequences, IEnumerable<ApplicationSection> qnaSections)
         {
-            var providerRoutes = await _roatpApiClient.GetApplicationRoutes();
-            var selectedProviderRoute = providerRoutes.FirstOrDefault(p => p.Id == providerRoute);
+            var selectedProviderRoute = "";
+            switch (providerRoute)
+            {
+                case 1: selectedProviderRoute = "Main provider";
+                    break;
+                case 2: selectedProviderRoute = "Employer provider";
+                    break;
+                case 3: selectedProviderRoute = "Supporting provider";
+                    break;
+                throw new ArgumentException(nameof(providerRoute));
```

Comment:
I can't see a `default:` on this switch statement

Reply:
Fixed